packages/toolpad-app/src/toolpad/ToolpadShell.tsx

```diff
@@ -107,7 +113,11 @@ function UpdateBanner() {
   return (
     <React.Fragment>
       {latestRelease && process.env.TOOLPAD_TARGET === 'CE' ? (
-        <Collapse in={!hideBanner}>
+        <Snackbar
+          open={!hideBanner}
+          onClose={handleDismissClick}
+          anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }}
```

Comment:
Should we use the `action` prop here?

Reply:
Something like this?:
```tsx
 ContentProps={{
            sx: { backgroundColor: (theme) => theme.palette.info.main },
          }}
          action={
            <Stack direction="row" sx={{ gap: 2 }}>
              <p>
                A new version <strong>{latestRelease.tag}</strong> of Toolpad is available.
              </p>
              <Button
                aria-label="update"
                color="inherit"
                endIcon={<OpenInNewIcon fontSize="inherit" />}
                component="a"
                target="_blank"
                href={DOCUMENTATION_URL}
              >
                Update
              </Button>
              <Button
                aria-label="view changelog"
                color="inherit"
                endIcon={<OpenInNewIcon fontSize="inherit" />}
                component="a"
                target="_blank"
                href={latestRelease.url}
              >
                View changelog
              </Button>
              <IconButton
                aria-label="close"
                color="inherit"
                size="small"
                onClick={handleDismissClick}
              >
                <CloseIcon fontSize="inherit" />
              </IconButton>
            </Stack>
          }
        />
  ```
  which results in (nicer than the original IMO): 
  ![Screenshot 2022-09-15 at 4 31 12 PM](https://user-images.githubusercontent.com/19550456/190387565-52e70730-3e18-4348-9802-c6add142ecf2.png)